### DISCUSSION TITLE
Error run capacitor iOS plugin on Simulator

### DISCUSSION THREAD

**Discussion by Rommel96** [ionic-team/capacitor#4181] [Q&A]
Hi people, 

I am trying run react app with `npx cap run ios`, but does not render the view and shows a blank screen. 

Web Inspector show this: 

![image](https://user-images.githubusercontent.com/52289346/107376034-6c17f180-6ab7-11eb-8d07-cf2d40fb48e3.png)

Dependencies:

`"@capacitor/android": "^3.0.0-beta.1"`
`"@capacitor/app": "^0.3.0"`
` "@capacitor/cli": "^3.0.0-beta.1"`
`"@capacitor/core": "^3.0.0-beta.1"`
` "@capacitor/ios": "^3.0.0-beta.1"`

In Android is good working.

_Please help and guide me if necessary_

**Comment by jcesarmobile** [comment]
Did you run npx cap sync after installing the plugin? Did it finish without problems?
Can you provide a sample app?

**Reply by Rommel96** [reply]
Sure,  npx cap sync finished without any problems: 

<img width="663" alt="image" src="https://user-images.githubusercontent.com/52289346/109733904-0f779600-7b8e-11eb-8b39-585d6db88444.png">
